dkg/paranet.py

```diff
@@ -83,6 +85,318 @@ def create(
             "operation": json.loads(Web3.to_json(receipt)),
         }
 
+    _add_paranet_curated_nodes = Method(BlockchainRequest.add_paranet_curated_nodes)
+
+    def add_curated_nodes(
+        self, ual: UAL, identity_ids: list[int]
+    ) -> dict[str, str | HexStr | TxReceipt]:
+        parsed_ual = parse_ual(ual)
+        knowledge_asset_storage, knowledge_asset_token_id = (
+            parsed_ual["contract_address"],
+            parsed_ual["token_id"],
+        )
+
+        self._add_paranet_curated_nodes(
+            knowledge_asset_storage,
+            knowledge_asset_token_id,
+            identity_ids,
+        )
+
+        return {
+            "paranetUAL": ual,
+            "paranetId": Web3.to_hex(
+                Web3.solidity_keccak(
+                    ["address", "uint256"],
+                    [knowledge_asset_storage, knowledge_asset_token_id],
+                )
+            ),
+            "operation": "Add curated nodes"
```

Comment:
Operations for these should be tx receipts, look into other functions

Reply:
Yes but in this case we do not have tx receipts. Perhaps I should just remove it from the return?

Reply:
We have receipts for all transactions, should have just add variable to store the returned receipts